src/cvi/__init__.py

```diff
@@ -14,7 +14,6 @@
 from .icvi import *
 from .modules import *
 
-
 class CVIOpts():
```

Comment:
add doc string

Reply:
This reminds me that I don't think that a CVI options class will be necessary. I'll remove it for now and make sure everything else has docstrings.